### PR TITLE
Optimise tweening by skipping get_sprite() and its assert

### PR DESCRIPTION
**Pull request by janisozaur**

### CODE REVIEW

File: src/openrct2.c [hunk 1]
```diff
@@ -75,7 +75,7 @@ EVP_MD_CTX *gHashCTX = NULL;
 int _finished;
 
 // Used for object movement tweening
-static struct { sint16 x, y, z; } _spritelocations1[MAX_SPRITES], _spritelocations2[MAX_SPRITES];
+static sprite_coords _spritelocations1[MAX_SPRITES], _spritelocations2[MAX_SPRITES];
```
Comment (IntelOrca) [inline src/openrct2.c]: Just use rct_xyz16... or are you going to tell me that packing will make it slower?

**Comment by IntelOrca** [comment]
Why does the compiler not optimise the assert out in release builds?

File: src/openrct2.c (hunk 1 above)
Comment (janisozaur) [inline src/openrct2.c]: Sorry, but `rct_xyz16` is already naturally aligned ;)

**Comment by janisozaur** [comment]
It may, but it would be nice for debug builds too. Though I'm not so certain about compiler really optimising it away, it does have to check access and the check is only done at 2 calls deep from caller.

**Comment by janisozaur** [comment]
updated

**Pull request merged**